services/dashboard/views/apps/apps.js

```diff
@@ -2,64 +2,78 @@
  * Licensed under the Apache License, Version 2.0
```

Comment:
Yes model layer should be first - with perhaps one or two services eg restapi. Controllers could be done in later tasks and probably piecemeal as was done in #952. I know there are concerns related to using scalajs-angular. What it does is fairly straightforward and low risk if we just limit its use to controllers and a few services especially since it has released 0.5.